articles/azure-functions/functions-reference-powershell.md

```diff
@@ -462,6 +462,21 @@ When you create a new PowerShell functions project, dependency management is ena
 
 When you update the requirements.psd1 file, updated modules are installed after a restart.
 
+If you want to use a specific version of a module, ie an older version of Az.Accounts then the one included in the included Az module, you need to add an import statement to the top of your profile.ps1 file like this:
```

Comment:
When I read this sentence, I may get an impression that all I need in this situation is to just add an import statement, as requrements.psd1 is not mentioned here. I think we should emphasize that _both_ are required, perhaps like this: "...you _also_ need to add an import statement..."